src/components/Nav.js

```diff
@@ -15,19 +22,70 @@ const Nav = () => {
     <Box
       direction="row"
       justify="between"
-      align="center"
+      align={['xsmall', 'small'].includes(size) ? 'center' : 'start'}
       width="xlarge"
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" color="plain" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box justify="between" align="start" alignSelf="center">
+        <RoutedAnchor
+          path="/"
+          icon={<GrommetIcon size="large" color="plain" />}
+          label={
+            !['xsmall', 'small'].includes(size) && (
+              <Text size="xlarge">grommet</Text>
+            )
+          }
+        />
+        {!['xsmall', 'small'].includes(size) && (
+          <Text>for {themeName === 'hpe' ? 'HPE' : 'open source'}</Text>
+        )}
+      </Box>
       <Box direction="row" gap="small">
         {!searchOpen && (
           <Box direction="row" gap="small">
+            {!['xsmall', 'small'].includes(size) && (
+              <Button
+                plain
+                onClick={() => {
+                  setThemeName(`${themeName === 'hpe' ? 'grommet' : 'hpe'}`);
+                }}
+              >
+                {({ hover }) => (
+                  <Box
+                    pad={{ vertical: 'small', horizontal: 'small' }}
+                    round="xlarge"
+                    background={hover ? 'active' : undefined}
+                    direction="row"
+                    gap="xsmall"
+                  >
+                    <Text>
+                      use for {themeName === 'hpe' ? 'open source' : 'HPE'}
+                    </Text>
+                    {themeName === 'grommet' && (
+                      <HpeIcon color="#01A982" size="medium" />
+                    )}
+                  </Box>
+                )}
+              </Button>
+            )}
+            {['xsmall', 'small'].includes(size) && (
+              <Menu
+                icon={<MoreIcon size="medium" />}
+                items={[
+                  {
+                    label: `use for ${
```

Comment:
```suggestion
                    label: `Use for ${
```

Reply:
We are repeating this string logic in the button and the menu, can we extract it to a single `const buttonLabel`?